plugins/core/src/main/kotlin/de/fayard/refreshVersions/core/RefreshVersionsCleanupTask.kt

```diff
@@ -11,11 +11,32 @@ import de.fayard.refreshVersions.core.internal.versions.VersionsPropertiesModel.
 import de.fayard.refreshVersions.core.internal.versions.readFromFile
 import de.fayard.refreshVersions.core.internal.versions.writeTo
 import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
 import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.options.Option
 import java.io.File
 
 open class RefreshVersionsCleanupTask : DefaultTask() {
 
+    @Input
+    @Optional
+    @Option(option = "enable", description = "Enable a feature flag")
+    var enableFlag: FeatureFlag? = null
+        set(value) {
+            field = value
+            if (value != null) FeatureFlag.userSettings.put(value, true)
+        }
+
+    @Input
+    @Optional
+    @Option(option = "disable", description = "Disable a feature flag")
+    var disableFlag: FeatureFlag? = null
+        set(value) {
+            field = value
+            if (value != null) FeatureFlag.userSettings.put(value, false)
+        }
```

Comment:
I think this should be in a separate PR, don't you?

Reply:
actually no, see comment :)